app/assets/stylesheets/comments.scss

```diff
@@ -734,6 +742,11 @@ a.header-link {
         }
       }
     }
+    &.hidden-comment {
```

Comment:
I don't think this should be faded. I feel like having it similar to the community standards violation design here could shame someone in the event of a hidden comment, which isn't what we're looking to do here.